packages/manager/apps/catalog/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useState, useEffect } from 'react';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsLink } from '@ovhcloud/ods-components/link/react';
+import { useLocation } from 'react-router-dom';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import FilterItem from './FilterItem';
+import LoadingFilterItem from './LoadingFilterItem';
+import './Filters.scss';
+import {
+  getFilterParamsFromUrl,
+  getAvailableCategoriesWithCounter,
+  getUniverses,
+  Product,
+} from '@/utils/utils';
+
+interface FiltersProps {
+  products: Product[];
+  setSearchValue: React.Dispatch<React.SetStateAction<string>>;
+  setLocalSearchValue: React.Dispatch<React.SetStateAction<string>>;
+  setSelectedCategories: React.Dispatch<React.SetStateAction<string[]>>;
+  setSelectedUniverses: React.Dispatch<React.SetStateAction<string[]>>;
+  onApply: () => void;
+}
+
+const Filters: React.FC<FiltersProps> = ({
+  products,
+  setSearchValue,
+  setLocalSearchValue,
+  setSelectedCategories: setParentSelectedCategories,
+  setSelectedUniverses: setParentSelectedUniverses,
+  onApply,
+}) => {
+  const [selectedCategories, setSelectedCategories] = useState<string[]>([]);
+  const [selectedUniverses, setSelectedUniverses] = useState<string[]>([]);
+  const [hasInteracted, setHasInteracted] = useState<boolean>(false);
+
+  const location = useLocation();
```

Comment:
should use `useSearchParams`

Reply:
Should be replace by this Hook in a next ticket 👍